<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6436" started-at="2018-07-20T13:44:23Z" finished-at="2018-07-20T13:44:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6436" started-at="2018-07-20T13:44:23Z" finished-at="2018-07-20T13:44:30Z">
      <class name="data_driver_framework.TestDDT1">
        <test-method status="PASS" signature="Setup()[pri:0, instance:data_driver_framework.TestDDT1@66a3ffec]" name="Setup" is-config="true" duration-ms="5964" started-at="2018-07-20T13:44:23Z" finished-at="2018-07-20T13:44:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="TestFireFox(java.lang.String, java.lang.String)[pri:0, instance:data_driver_framework.TestDDT1@66a3ffec]" name="TestFireFox" duration-ms="455" started-at="2018-07-20T13:44:29Z" data-provider="TestDataFeed" finished-at="2018-07-20T13:44:30Z">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
Data provider mismatch
Method: TestFireFox([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(org.apache.poi.xssf.usermodel.XSSFCell) abdul,(org.apache.poi.xssf.usermodel.XSSFCell) abdul]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
Data provider mismatch
Method: TestFireFox([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(org.apache.poi.xssf.usermodel.XSSFCell) abdul,(org.apache.poi.xssf.usermodel.XSSFCell) abdul]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:45)
at org.testng.internal.Parameters.injectParameters(Parameters.java:796)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:982)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestFireFox -->
        <test-method status="PASS" signature="QuitTC()[pri:0, instance:data_driver_framework.TestDDT1@66a3ffec]" name="QuitTC" is-config="true" duration-ms="1" started-at="2018-07-20T13:44:30Z" finished-at="2018-07-20T13:44:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitTC -->
      </class> <!-- data_driver_framework.TestDDT1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
